src/types.ts

```diff
@@ -33,54 +33,85 @@ type ScreenshotState = {
   screenshots: ReadonlyArray<Screenshot>
 }
 
-type TwitterHandleStatus = 'NEW' | 'IN_PROGRESS' | 'DONE'
-
-type DomainTwitterHandle = {
-  status: TwitterHandleStatus
-  handle: string | null
-}
-
 type EditingScreenshotState = {
   color: string
   screenshot: Screenshot
 }
 
 type FeedbackState = {
+  isTweeting: boolean
```

Comment:
Why did you move `isTweeting` inside the feedback?

Reply:
So that this was a one-liner

```
return updateActiveFeedback(state, () => ({ isTweeting: true }))
```